api-reference/50 Common/Object Structures/dxSchedulerAppointment/recurrenceException.md

```diff
@@ -4,12 +4,19 @@ type: String
 ---
 ---
 ##### shortDescription
-Specifies exceptions for the current recurring appointment.
+Specifies the start date and time of one or more appointments to delete from a series. Requires [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule) to be set.
 
 ---
-To define an exception for a recurring appointment, specify the **recurrenceException** field in the appointment object. This field enables you to specify several appointments to exclude. Each appointment is specified by the start date and time. The start time of each excluded appointment should strictly match the start time of the initial appointment.
 
-[note]This option makes sense only if the appointment [recurrenceRule](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/recurrenceRule.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule') is specified.
+In the example below, the appointment takes place daily except Fabruary 22 and 23, 2021. Note that **recurrenceException** sets time in UTC as well as [startDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate) and [endDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate):
+
+    {
+        text: 'Daily planning',
+        startDate: new Date('2021-02-20T07:00:00.000Z'),
+        endDate: new Date('2021-02-20T08:00:00.000Z'),
+        recurrenceRule: 'FREQ=DIALY',
+        recurrenceException: '20210222T070000Z,20210223T070000Z'
+    }
```

Comment:
I suggest you to write a full snippet for this for all platforms. Fix also 'FREQ=DIALY', please.